test/algorithms/multi_length.cpp

```diff
@@ -8,15 +8,8 @@
 
 #include <algorithms/test_length.hpp>
 
-#include <boost/geometry/multi/multi.hpp>
-
 #include <boost/geometry/geometries/geometries.hpp>
 #include <boost/geometry/geometries/point_xy.hpp>
-#include <boost/geometry/multi/geometries/multi_linestring.hpp>
-
-#include <boost/geometry/io/wkt/wkt.hpp>
```

Comment:
OK for me. It was probably originally included as ".../multi/io/wkt" which is why it was still there. Having it in test_length (etc) is OK